addons/website/views/snippets/snippets.xml

```diff
@@ -445,5 +440,54 @@
                 data-name="scroll_button_opt"/>
         </we-row>
     </div>
+
+    <!-- Tabs -->
+    <div data-js="NavTabs" data-selector="section" data-target=".s_tabs_wrap">
+        <we-button data-add-tab=""
+                    data-no-preview="true"
+                    class="fa fa-fw fa-plus"
+                    title="Add Tab"
+                    aria-label="Add Tab"/>
+        <we-button data-remove-tab=""
+                    data-name="remove_tab_opt"
+                    data-no-preview="true"
+                    class="fa fa-fw fa-minus"
+                    title="Remove Tab"
+                    aria-label="Remove Tab"/>
+    </div>
+    <div data-js="NavTabsStyle" data-selector="section" data-target=".s_tabs_wrap">
+        <we-select string="Style">
+            <we-button data-set-style="tabs" data-name="tabs_opt" data-trigger="horizontal_opt">Tabs</we-button>
+            <we-button data-set-style="pills" data-name="pills_opt" data-trigger="tabs_color_opt" data-trigger-value="">Buttons</we-button>
+        </we-select>
+        <we-colorpicker string="Tabs color"
+            data-dependencies="tabs_opt"
+            data-name="tabs_color_opt"
+            data-select-style="true"
+            data-css-property="background-color"
+            data-color-prefix="bg-"/>
+        <we-select string="Direction" data-dependencies="pills_opt">
+            <we-button data-set-direction="horizontal" data-name="horizontal_opt">Horizontal</we-button>
+            <we-button data-set-direction="vertical" data-trigger="left_alignment_opt">Vertical</we-button>
+        </we-select>
+        <we-select string="Alignment" data-apply-to="ul.nav" data-dependencies="horizontal_opt">
+            <we-button data-select-class="" data-name="left_alignment_opt">Left</we-button>
+            <we-button data-select-class="justify-content-center">Center</we-button>
+            <we-button data-select-class="justify-content-end">Right</we-button>
+        </we-select>
+        <we-select string="Fill and justify" data-apply-to="ul.nav" data-dependencies="horizontal_opt">
+            <we-button data-select-class="">Regular</we-button>
+            <we-button data-select-class="nav-fill">Fill</we-button>
+            <we-button data-select-class="nav-justified">Justified</we-button>
+        </we-select>
+        <we-divider/>
+        <we-select string="Transition">
+            <we-button data-select-class="">No Slide Effect</we-button>
+            <we-button data-select-class="s_tabs_slide_up">Slide Up</we-button>
+            <we-button data-select-class="s_tabs_slide_down">Slide Down</we-button>
+            <we-button data-select-class="s_tabs_slide_left">Slide Left</we-button>
+            <we-button data-select-class="s_tabs_slide_right">Slide Right</we-button>
+        </we-select>
+    </div>
```

Comment:
The old `NavTabs` option was here because it was generic, targeting standard bootstrap classes. It is now specific to the snippet, targeting `s_tabs_wrap`. My guess is that the first part NavTabs could stay generic and that the second part NavTabsStyle must be specific with an options.js file etc. No?

Reply:
At first, the idea was to reuse this snippet for a new comparisons snippet. So I made it this way but as the comparisons snippet has been canceled you're right it is better to do it like that